docs/3.x.x/en/guides/authentication.md

```diff
@@ -316,14 +320,13 @@ As for backend, we have a `switch...case` where we need to put our new provider
 Add the corresponding translation into: `packages/strapi-plugin-users-permissions/admin/src/translations/en.json`
 
 ```js
-  "PopUpForm.Providers.discord.providerConfig.redirectURL": "The redirect URL to add in your Discord application configurations",
+  "PopUpForm.Providers.discord.providerConfig.redirectURL": "The redirect URL to add in your Discord config",
```

Comment:
Can you revert this update please.
If you check all translations files it's:

`"PopUpForm.Providers.discord.providerConfig.redirectURL": "The redirect URL to add in your Discord application configurations",`

`"PopUpForm.Providers.facebook.providerConfig.redirectURL": "The redirect URL to add in your Facebook application configurations",`

Reply:
Then I will be ready to merge your PR :)